dlairflow/util.py

```diff
@@ -10,17 +10,28 @@
 import os
 
 
-def user_scratch():
+def user_scratch(owner):
     """A standard, per-user scratch directory.
 
     This function simply returns a path. It does not guarantee the directory exists.
+    The environment variable :envvar:`DLAIRFLOW_SCRATCH_ROOT` must be set.
+
+    Parameters
+    ----------
+    owner : :class:`str`
+        The owner of a DAG. This can be obtained from, *e.g.* ``dag.owner``.
```

Comment:
If you make the change to the argument name then this text could be updated to something along the lines of:
```suggestion
    user_key : :class:`str`
        The key associated with the a user's scratch space. This can be any arbitrary string such as a single DAG owner *e.g.* ``dag.owner``.
```

Reply:
Agreed, done.